ga4gh/backend.py

```diff
@@ -782,23 +782,34 @@ def runSearchRequest(
         self.endProfile()
         return responseString
 
-    def runListReferenceBases(self, id_, requestArgs):
+    def runListReferenceBases(self, id_, requestJson):
         """
         Runs a listReferenceBases request for the specified ID and
         request arguments.
         """
-        compoundId = datamodel.ReferenceCompoundId.parse(id_)
+        # In the case when an empty post request is made to the endpoint
```

Comment:
Yes, I believe it is still a `ListReferenceBases` request. Do you think it is a good idea for endpoints that have all optional fields (like datasets/search) require a JSON be posted? Range is optional in this request. 

I always found this choice awkward and inconsistent with other APIs I have used. I have to make an empty JSON in order to list datasets in the reference server.